extensions/Worker.Extensions.Http.AspNetCore.Analyzers/Worker.Extensions.Http.AspNetCore.Analyzers.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <VersionPrefix>1.0.0</VersionPrefix>
+        <OutputType>Library</OutputType>
+        <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+        <IncludeBuildOutput>false</IncludeBuildOutput>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <PackageId>Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore.Analyzers</PackageId>
```

Comment:
Since we are not planning to publish a package from this project, Shall we consider setting `<IsPackable>false</IsPackable>` ?

Also, Should we remove the other properties which are used for a project which gets packed. Ex: PackageId, Description, VersionPrefix et..

Reply:
I tried doing this but the analyzer is not working as expected. Some examples I found seem to be packing it, not sure if this is a requirement for Analyzer. Happy to debug this with you but don't think we should block on this.